gradle/wrapper/gradle-wrapper.properties

```diff
@@ -3,4 +3,4 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.1-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.5.1-all.zip
```

Comment:
Is this a must?

Reply:
No. It's ok to have this feature with gradle 4.1. However, the plugin and kotlin version(1.1.3) is pretty old in gradle 4.1. From gradle 4.5, the built-in kotlin version is upgraded to 1.2.0.

Reply:
I think we should update this. Just maybe not before release so maybe merge this after we build beta rc.